tracer/test/Datadog.Trace.Tests/DistributedTracer/AutomaticTracerTests.cs

```diff
@@ -96,5 +97,19 @@ public void GetDistributedTrace()
 
             automaticTracer.GetDistributedTrace().Should().BeNull();
         }
+
+        [Fact]
+        public void RuntimeId()
```

Comment:
Similar to `ManualTracerTests.IsChildTracer`, can you add a `AutomaticTracerTests.IsNotChildTracer` test?

Reply:
Good catch, added in https://github.com/DataDog/dd-trace-dotnet/pull/2172/commits/2190b6fe2e868c3f09fd9f69ea88429ec0d5d200